.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  release:
+    types: [created]
+  workflow_dispatch:
+
+jobs:
+  build_and_test:
+    name: Install and test
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade tox tox-wheel coveralls
```

Comment:
Use minversion and requires flags to bootstrap tox, tox-wheel and tox-gh-actions

https://tox.readthedocs.io/en/latest/config.html#conf-requires

Reply:
Done. Added `minversion` and tox-related dependencies to the corresponding section in `pyproject.toml`. Also, use `tox-gh-actions` to manage different versions of Python. I think `tox-gh-actions` is an overkill to this project, but I still used it because it's recommended by Jazzband guidelines.